### PR TITLE
Require class decorator

### PR DESCRIPTION
**Pull request by mambocab**
- Simplifies the `@require` decorator and makes it work on classes.
- Moves `@since` and `@require` decorations from methods to class level on `TestGlobalIndexes`.

### CODE REVIEW

File: tools.py [hunk 1]
```diff
@@ -214,14 +214,9 @@ def wrapped(obj):
...
-    def __call__(self, f):
-        def wrapped(obj):
-            obj.skip("require " + self.msg)
-            f(obj)
-        wrapped.__name__ = f.__name__
-        wrapped.__doc__ = f.__doc__
-        return wrapped
\ No newline at end of file
+def require(msg):
+    """Skips the decorated class or method with a mess age about which Jira
```
Comment (ptnapoleon) [inline tools.py]: Fix the typo in message.

**Comment by ptnapoleon** [comment]
Does require still work on methods as well?

**Comment by ptnapoleon** [comment]
If so, +1 other than the typo.

**Comment by mambocab** [comment]
> Does require still work on methods as well?

Yep. On [this test](https://github.com/riptano/cassandra-dtest/blob/master/user_types_test.py#L677):

```
CASSANDRA_DIR=~/cstar_src/cassandra nosetests -vs user_types_test.py:TestUserTypes.udt_subfield_test
udt_subfield_test (user_types_test.TestUserTypes) ... SKIP: require 7423

----------------------------------------------------------------------
Ran 1 test in 0.001s

OK (SKIP=1)
```

**Comment by mambocab** [comment]
Oh, and for the record, this was first brought up [here](https://github.com/riptano/cassandra-dtest/pull/176#issuecomment-77231022).

**Comment by ptnapoleon** [comment]
Sgtm. Feel free to merge.

**Pull request merged**